### ISSUE TITLE
platform web opens wrong URL

### ISSUE THREAD

**Issue by JGrubb** [platformsh/platformsh-cli#518]
Currently in 3.5.1 `platform web` opens a API endpoint instead of the project admin.

```
☺  platform web
Opened: https://us.platform.sh/api/projects/kde3ldrxwiopg/environments/master
```

which is obviously a json response - 

```json
{
  status: "error",
  message: "Access was denied to this resource.",
  code: 403,
  detail: "Unauthorized: document_retrieve_view failed permission check",
  title: "Forbidden"
}
```

**Comment by pjcdawkins** [comment]
Yes, silly mistake... this will be fixed in the next release via ba40a7be40cbf5a020647b986abc823a87944ab7

**Comment by pjcdawkins** [comment]
Fixed in v3.5.2
